internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -169,3 +225,33 @@ func parseExpose(line string) []portConfig {
 	}
 	return ports
 }
+
+func parseHealthCheck(line string) (healthCheck, error) {
```

Comment:
Should we return `*healthcheck, error` instead? if the HEALTHCHECK is "NONE" then I think returning nil instead of an empty healthcheck makes more sense.

Reply:
Yep!